# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/html/HTMLElementAccessibles.cpp
# Commit: ffbb26d2a3c1
# Full Hash: ffbb26d2a3c186748ef9367f6d265cedffbe6a69
# Author: Morgan Rae Reschenberg <mreschenberg@mozilla.com>
# Date: 2022-08-02 09:41:16
# Regressor Bug: 1774043
# File Overlap Count: 1
# Description:
#   Bug 1774043: [Part 1] Add cache domains, caching logic for LABEL_FOR and LABELLED_BY relations r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152100
# ==============================================================================

diff -r e0b787859230 -r ffbb26d2a3c1 accessible/html/HTMLElementAccessibles.cpp
--- a/accessible/html/HTMLElementAccessibles.cpp	Tue Aug 02 04:59:08 2022 +0300
+++ b/accessible/html/HTMLElementAccessibles.cpp	Tue Aug 02 02:51:03 2022 +0000
@@ -66,6 +66,7 @@
                                                aModType, aOldValue, aOldState);
 
   if (aAttribute == nsGkAtoms::_for) {
+    mDoc->QueueCacheUpdate(this, CacheDomain::Relations);
     SendCache(CacheDomain::Actions, CacheUpdateType::Update);
   }
 }
